### DISCUSSION TITLE
Pull Requests Ignored

### DISCUSSION THREAD

**Discussion by toasted-time-traveller** [BishopFox/sliver#1803] [Q&A] [answered]
I have noticed some pull requests are not being accepted or closed/denied including mine. Why is this? I know tone can be difficult to get across via text so please see this as a me asking a genuine question and not portraying annoyance. I am also very new to Github as I previously stored all of my work locally so I have a lot to learn and the fact these pull requests, including mine, are being left may well be being left for a specific reason.

**Comment by moloch--** [comment, marked as answer]
We only have so much free time, we're not paid (i.e., we're not allocated specific time) to work on Sliver and the amount of time we can spend on the project is largely subject to our professional workloads.

**Reply by toasted-time-traveller** [reply]
Thank you to you both (moloch + rkervella) for such a quick reply, I fully understand. There were some PR's going back over a year that were still open and it was mainly those that made me wonder and then think mine was going to be left as well. Once again, thank you for replying so quickly and accepting the merge. I hope I can contribute some worthwhile code to sliver in the future.

**Comment by rkervella** [comment]
> I have noticed some pull requests are not being accepted or closed/denied including mine. Why is this?

Reviewing pull requests takes time. However, when we close a PR, we usually comment as to why we're closing it.
